flash/core/data/utils.py

```diff
@@ -70,8 +73,7 @@ def download_data(url: str, path: str = "data/", verbose: bool = False) -> None:
     # __author__  = "github.com/ruxi"
     # __license__ = "MIT"
 
-    Examples
-    ________
+    Examples:
```

Comment:
We tend to use the underline everywhere. Any particular reason for this change?

Reply:
not really... sorry :)